src/services/user-auth.ts

```diff
@@ -54,7 +54,7 @@ module UserAuth{
     export function authenticateUser(req:Restify.Request, em:string, pass:string) : Promise<number> {
 
         return new Promise<number>((resolve, reject) => {
-            userSer.getUserByEmail(em)
+            userSer.getUserByEmail(em.toLowerCase())
```

Comment:
If the user service is lowercasing any incoming email address, is it necessary to pass it in lowercased?